cli/src/mdastToRst.ts

```diff
@@ -238,6 +238,11 @@ const visitors: {
     if (value === undefined) {
       return;
     }
+    if (value.endsWith(" - ")) {
+      // rst does not like the "- ``" character sequence. Don't ask me why.
+      c.add(`\`\`${value.slice(0, -3)}\`\`: `);
```

Comment:
It would also be clean to add an escaper to the add() calls below. What happens if the value is multiline that also ends with " - "? This would not handle the "weird case" below.